project/F84064014/README.rst

```diff
@@ -0,0 +1,101 @@
+======================================================================
+A Simple End-to-End NN Framework for Deep Learning
+======================================================================
+
+Basic Information
+=================
+
+Deep Learning is now widely used to solve a lot of problem from many different field, 
+such as computer vision, data classification, natural language processing, etc., this
+project we will implement a End-to-End Deep Learning Framework and accelerate it with C++
+
+Problem to Solve
+================
+
+In the project we try to make deep learning faster and easier
+
+Deep Learning is a powerful tool even for people who never write code before, however, 
+build a deep learning model is not a walk in the park, we hope to build a framework
+that user have to care only about which dataset they are going to use, how many layers they
+want, which loss function they want, then they could train a powerful model on their task.
+
+Neaural Network is a important part of deep learning model. Training a neural network
+may cost a lot of time especially when there is no gpu available, thus some component
+would be written in C++ to make it faster
+
+Prospective Users
+=================
+
+People who wants to use deep learning method but have no idea of programming or people
+who just start to learn about deep learning.
+
+System Architecture
+===================
+
+A main class FNN will be written in C++ and connect to python script with pybind11.
+In python script FNN should be a callable class with some different function. Some
+fucntion like dense layer, activation are subclass defined inside FNN object.
+
+Each FNN maintain a list(doubly linked) of layer object. All layers are made of a
+2d double array with size=(input_size, output_size), and followed by an activation
+fucntion.
+
+API Description
+===============
+
+FNN class includes complete implementation of a Feed Forward Neaural Network, user only
+have to initialize model and train in python script. model configuration is
+determined by a string, each line of string start with layer name and followed with
+their parameter
+
+API function
+- FNN.__init__(): initialize parameter(layer) and flag
```

Comment:
It's OK to have the first prototype to do the initialization in the python `__init__`.  But oftentimes we may find a better place for initializing the data.  Let's see how it goes.